API.md

```diff
@@ -23,8 +23,10 @@ option | values | function
 --- | --- | ---
 drawing | boolean | The ability to draw and delete features - default: `true`
 keybindings | boolean | Keyboard shortcuts for drawing - default: `true`
+touchEnabled | boolean | If true, enables touch interaction - default: `true`
 boxSelect | boolean | If true, shift + click to features. If false, click + select zooms to area - default: `true`
 clickBuffer | number | On click, include features beyond the coordinates of the click by clickBuffer value all directions - default: `2`
+touchBuffer | number | On touch, include features beyond the coordinates of the touch by touchBuffer value all directions - default: `25`
```

Comment:
Added a bit more explanation, but I'm afraid I can't back it up with science. It just seemed to feel right on the devices i tried.